### ISSUE TITLE
Report broken site link

### ISSUE THREAD

**Issue by orschiro** [ZachSaucier/Just-Read#13]
Dear @ZachSaucier,

What do you think of adding a link `Report broken site` to the rendered preview that forwards to Github new issue creation with the mentioning of the broken site link?

![image](https://cloud.githubusercontent.com/assets/170947/19110447/46fb98b2-8afa-11e6-9a8f-2d9a726c3b6d.png)

**Comment by ZachSaucier** [comment]
This is a good idea! 

The only problem with it that I can think of right now is that a lot of sites will share the same core issue. One example of a core issue that I'm already aware of is sites that have a lot of text not inside of paragraph tags but have a single paragraph tag in the description or something, which causes Just Read to just select that paragraph. 

It'd be more useful to report each site to a particular division based on the issue that is causing it (which usually means using the developer tools and poking around a bit, which is obviously more than what most users can/want to do, but might be able to be automated some based on the error).

___

But it might be interesting to at least try out for a bit. I need to think a bit as to what method of reporting would be best. 

One way I imagine it could work is to have a "Report this site as broken" link which, when clicked, sends the URL to a google sheet (maybe even with the most recent console error message?), but I'm not sure how that's possible. I'll continue to explore this option a bit more.

The other way I can imagine is linking to a single GitHub issue (or an issue pointing towards other issues for each core problem) so that users can post there so I don't get dozens of different issues all with single bugs, perhaps caused by the same issue.

**Comment by orschiro** [comment]
> The other way I can imagine is linking to a single GitHub issue (or an
issue pointing towards other issues for each core problem) so that users
can post there so I don't get dozens of different issues all with single
bugs, perhaps caused by the same issue.

I like this simple and clean idea!

**Comment by ZachSaucier** [comment]
After thinking about it some more, I decided that a GitHub issue listing the bug types is too similar to the functionality that GitHub has [using filters](https://github.com/ZachSaucier/Just-Read/issues?utf8=%E2%9C%93&q=is%3Aissue%20label%3Abug%20). 

So, I decided to link to this page using a "Report an error" link at the bottom of the page. This should be live in version 1.0.5, which is being uploaded now.

Thanks for the feature request!
